src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -89,7 +89,7 @@ function ChartValuesView({
 }: ChartValuesViewType) {
     const history = useHistory()
     const { url } = useRouteMatch()
-    const { chartValueId } = useParams<{ chartValueId: string }>()
+    const { chartValueId, presetValueId } = useParams<{ chartValueId: string; presetValueId: string }>()
```

Comment:
Can we add default values for chartValueId, presetValueId ?

Reply:
These are URL params and we don't need default values as we already have some checks around the same wherever they're used.